site/archetypes/concept.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+---
+
+## Overview
+
+<!-- 
+Briefly introduce the concept. Keep it high-level and user-friendly.
+What is it? Why does it matter? What problem does it solve?
+Example: An `HTTPRoute` defines how HTTP traffic is routed to backend services in a Kubernetes environment.
+-->
+
+## When to Use This
+
+<!-- 
+Describe common use cases. When would someone need this concept?
+List scenarios where understanding this will help the user make better architectural or operational decisions.
+-->
+
```

Comment:
should we add an optional Before you Begin, with hyperlinks to prerequisite concepts ?